### PR TITLE
Fix golangci issues for container stuff

### PR DESCRIPTION
**Pull request by vboulineau**
### What does this PR do?
https://datadoghq.atlassian.net/browse/CONT-705

Fix all golangci issues for the folders we were flagged on.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: pkg/util/kubernetes/hostinfo/no_apiserver.go [hunk 1]
```diff
@@ -7,6 +7,6 @@
 
 package hostinfo
 
-func apiserverNodeLabels(nodeName string) (map[string]string, error) {
+func apiserverNodeLabels(nodeName string) (map[string]string, error) { //nolint
```
Comment (hush-hush) [inline pkg/util/kubernetes/hostinfo/no_apiserver.go]: Could you add a comment after the `//nolint` explaining why we don't lint it ?

**Review by hush-hush — CHANGES_REQUESTED** [review]
Added a small nitpick, looks great otherwise !

File: pkg/util/kubernetes/hostinfo/no_apiserver.go (hunk 1 above)
Comment (vboulineau) [reply pkg/util/kubernetes/hostinfo/no_apiserver.go]: Yeah the method was never used, but I wanted to keep it for consistency with other no_*.go, but in fact I think it's useless, I removed the file thus removing the `//nolint` :D

**Review by hush-hush — DISMISSED** [review]

**Comment by vboulineau** [comment]
@hush-hush I've removed my changes about GCGO and CircleCI VM Size (8GB RAM) as you've done the loop per package in inv tasks. Though locally I still encountered swapping that make the VM unresponsive from time to time just linting ./pkg with 4GB RAM.

I'm afraid it will randomly fail when people remove excluded packages, WDYT?

**Comment by hush-hush** [comment]
@vboulineau If we could increase the VM memory size it would be much better indeed. I just push a very quick fix to unlock master.

**Review by hush-hush — APPROVED** [review]

**Review by mfpierre — APPROVED** [review]
:shipit:

**Pull request merged**